udata_front/theme/gouvfr/assets/js/components/search/OrganizationSearch/OrganizationSearch.vue

```diff
@@ -36,15 +37,16 @@
           :id="option.id"
           role="option"
           :aria-selected="isSelected(option.id)"
+          @mousedown.prevent="preventCollapse"
```

Comment:
in MenuSearch.vue, the `@mousedown.prevent` is on the `<div>` and without a `preventCollapse` function. Is there a reason for changes this here ?